docs/ui/customization/layout.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Layout
+description: Managing the layout of UI Components
+filterKey: framework
+---
+
+### Managing Layout with CSS
```

Comment:
The top-level customization section is meant for component-agnostic customization documentation. Instead of using "layout" here, maybe we can just highlight how you can select the top level tag to customize css. Similar to Ionic docs: https://ionicframework.com/docs/theming/css-variables#component-variables

Reply:
Ok cool, so I can remove the `filter` field here?

Reply:
Ah actually I see, instead of calling it "layout" it can be something like "Component Variables"?